src/lib/site-import/status.ts

```diff
@@ -235,12 +259,12 @@ ${ maybeExitPrompt }
 
 	progressTracker.startPrinting( setSuffixAndPrint );
 
-	const getResults = () =>
+	const getResults = (): Promise< string | Job | PrimaryDomainSwitchJob > =>
 		new Promise( ( resolve, reject ) => {
-			const checkStatus = async () => {
+			const checkStatus = async (): Promise< unknown > => {
 				let status;
 				try {
-					status = await getStatus( api, app.id, env.id );
+					status = await getStatus( api, app.id ?? -1, env.id ?? -1 );
```

Comment:
In theory, according to GQL definitions, `id` could be `null` or `undefined`. I chose -1 to make the request fail.